.editorconfig

```diff
@@ -47,3 +47,6 @@ indent_size = 4
 tab_width = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+dotnet_diagnostic.IDE0017.severity = none # IDE0017: Simplify object initialization
```

Comment:
I don't know why no-one in the team likes this rule xD

Reply:
It fails when any of the initializing code has null ref and does not tell you which one it was.